torch/distributions/logistic_normal.py

```diff
@@ -0,0 +1,46 @@
+import torch
+from torch.distributions import constraints
+from torch.distributions.normal import Normal
+from torch.distributions.transformed_distribution import TransformedDistribution
+from torch.distributions.transforms import ComposeTransform, ExpTransform, StickBreakingTransform
+from torch.distributions.utils import _sum_rightmost
+
+
+class LogisticNormal(TransformedDistribution):
+    r"""
+    Creates a logistic-normal distribution parameterized by
+    `mean` and `std` where::
+
+        X ~ LogisticNormal(loc, scale)
+        Y = log(X / (1 - X)) ~ Normal(loc, scale)
+
+    Example::
+
+        >>> m = LogisticNormal(torch.Tensor([0.0]), torch.Tensor([1.0]))
```

Comment:
Could you add a higher dimensional example, say with 3 inputs and 4 outputs? It's not clear to me from this documentation what the `loc` and `scale` parameters mean.